its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java

```diff
@@ -17,38 +17,34 @@
 package com.sonar.python.it.plugin;
 
 import com.sonar.orchestrator.build.SonarScanner;
-import com.sonar.orchestrator.junit5.OrchestratorExtension;
 import java.io.File;
 import java.util.List;
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 import org.sonarqube.ws.Issues;
 
 import static com.sonar.python.it.plugin.TestsUtils.issues;
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
 class NotebookPluginTest {
 
   private static final String PROJECT_KEY = "ipynb_json_project";
 
   @RegisterExtension
-  public static final OrchestratorExtension ORCHESTRATOR = TestsUtils.ORCHESTRATOR;
+  public static final ConcurrentOrchestratorExtension ORCHESTRATOR = TestsUtils.ORCHESTRATOR;
 
-  @BeforeAll
-  static void startServer() {
+  @Test
+  void test() {
```

Comment:
Running the test locally, the issues do look legit. We could update the assertions (I believe asserting an empty list of issue was a mistake).
We can do that here or in a separate PR, as you prefer.

Reply:
I'll change the assertion. I think it should be doable in this PR